pkg/service/v0/service.go

```diff
@@ -29,10 +34,81 @@ func (s Hello) Greet(ctx context.Context, req *v0proto.GreetRequest, rsp *v0prot
 		return ErrMissingName
 	}
 
-	rsp.Message = fmt.Sprintf(
-		"Hello %s",
-		req.Name,
-	)
+	phrase := getGreetingPhrase(ctx)
+	rsp.Message = fmt.Sprintf(phrase, req.Name)
 
 	return nil
 }
+
+func getGreetingPhrase(ctx context.Context) string {
+	ownAccountUUID := ctx.Value(middleware.UUIDKey)
+	if ownAccountUUID != nil {
+		// request to the settings service requires to have the account uuid of the authenticated user available in the context
+		request := &settings.GetSettingsValueRequest{
+			Identifier: &settings.Identifier{
+				Extension:   "ocis-hello",
+				BundleKey:   "greeting",
+				SettingKey:  "phrase",
+				AccountUuid: ownAccountUUID.(string),
+			},
+		}
+
+		// TODO this won't work with a registry other than mdns. Look into Micro's client initialization.
```

Comment:
We should open an issue to capture this, and ideally test this with a different registry